Documentation/DataSources/DataSources.md

```diff
@@ -149,6 +150,29 @@ To avoid automatically making calls to un-trusted hosts when using templates, th
 
 This provider supports [JSON Path](../Transformations/JSONPath.md).
 
+## Azure RBAC
+The Azure RBAC provider allows you to check permissions on resources. It is most commonly used to export the resulting checks to a parameter and display an error message or disable a button if there's no permission. Simple JSON arrays or objects will automatically be converted into grid rows and columns or text with a 'hasPermission' column with either true or false. The permission is checked on each resource and then either 'or' or 'and' to get the result. The [operations or actions](https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations) can be a string, an array, or json with action array.
+
+**String:**
+```
+"microsoft.insights/workbooks/read"
+```
+
+**Array:**
+```
+["microsoft.insights/workbooks/read", "microsoft.insights/workbooks/write"]
+```
+
+**Action list:**
+```
+{
+  "actions": [
```

Comment:
does this json object usually need to have anything else other than actions array? I wonder if we can just omit this part for simplicity and have users only use the string or Array as the input?

Reply:
that's a good point and until i noticed that https://docs.microsoft.com/en-us/rest/api/authorization/permissions/list-for-resource supported this and i added it. we could just remove it from the docs for now

Reply:
Do we support notActions for the RBAC provider in workbooks?